sqlglot/tokens.py

```diff
@@ -127,6 +128,7 @@ class TokenType(AutoName):
     ANY = auto()
     APPLY = auto()
     ARRAY = auto()
+    ARRAY_OVERLAPS = auto()
```

Comment:
Added this instead of using `OVERLAPS` because there's already an `OVERLAPS` binary operator in postgres.

Reply:
maybe it should be damp? double amp